Misc/NEWS.d/next/Build/2019-10-13-16-18-24.bpo-38468.buCO84.rst

```diff
@@ -0,0 +1,3 @@
+Use `getvar` for all still existing `sysconfig.get_config_var()` calls.
+The affected file is `Python3.9-config.in`.
+Patch by Joannah Nanjekye

```

Comment:
python-config.in is compiled to python-config and then installed as python3.9-config. If I recall correctly, it's only used on macOS. On Linux, the shell version is preferred: Misc/python-config.sh.in.

```suggestion
Misc/python-config.in now uses `getvar` for all still existing `sysconfig.get_config_var()` calls.
Patch by Joannah Nanjekye 
```